listenbrainz/webserver/templates/base.html

```diff
@@ -17,7 +17,8 @@
       <link href="{{ url_for('static', filename='css/main.less') }}" rel="stylesheet/less" type="text/css" />
       <script src="{{ url_for('static', filename='js/lib/less.min.js') }}"></script>
     {% else %}
-      <link href="{{ url_for('static', filename='css/main.css') }}" rel="stylesheet" media="screen" />
+      <!-- The css file has a .less extension in the manifest file entry (due to its original name in wbepack entry) -->
```

Comment:
use a flask comment: 
`{# The css file has a ...  #}`
so that we don't render it in the final template

Reply:
spelling: wbepack -> webpack